### DISCUSSION TITLE
Ram usage as unknown when LXC is OFF

### DISCUSSION THREAD

**Discussion by snp88** [dougiteixeira/proxmoxve#277] [Feature request] [answered]
Hi 
First at all your integration is great :)

i have small request :) 

Can we show Ram usage as 0% instead of unknown value when the LXC is off 
![image](https://github.com/dougiteixeira/proxmoxve/assets/46537540/95176812-f8f9-489f-b79a-6c78061d895a)

0 is not far from true right :) when the lxc is off it don't use any RAM.
I think 0 value is more elegant 
Let me know what you think

**Comment by dougiteixeira** [comment, marked as answer]
Added in #306. It will be available in the next version (3.4.2).
